build_web_compilers/lib/src/dev_compiler_bootstrap.dart

```diff
@@ -258,26 +265,54 @@ String _entryPointJs(String bootstrapModuleName) => '''
   var mainUri = _currentDirectory + "$bootstrapModuleName";
 
   if (typeof document != 'undefined') {
-    var el = document.createElement("script");
+    var el;
+
+    // Load stack trace mapper if not already loaded
+    var mapperId = 'dart-stack-trace-mapper';
+    if (!document.querySelector('#' + mapperId)) {
+      el = document.createElement("script");
+      el.id = mapperId;
+      el.defer = true;
+      el.async = false;
+      el.src = mapperUri;
+      document.head.appendChild(el);
+    }
+
+    // Load require.js if not already loaded
+    var requirejsId = 'dart-requirejs';
```

Comment:
Not sure if these IDs are specific enough to avoid any potential collisions, thoughts?

Reply:
I think this is fine 👍